radicle-cli/src/commands/patch.rs

```diff
@@ -24,79 +24,93 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad patch [<option>...]
+    rad patch
+    rad patch open [<option>...]
+    rad patch update <id> [<option>...]
 
-Create options
+Create/Update options
 
-    -u, --update [<id>]        Update an existing patch (default: no)
+        --[no-]confirm         Don't ask for confirmation during clone
         --[no-]sync            Sync patch to seed (default: sync)
         --[no-]push            Push patch head to storage (default: true)
     -m, --message [<string>]   Provide a comment message to the patch or revision (default: prompt)
         --no-message           Leave the patch or revision comment message blank
 
 Options
 
-    -l, --list                 List all patches (default: false)
         --help                 Print help
 "#,
 };
 
-#[derive(Debug)]
-pub enum Update {
-    No,
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum OptPatch {
     Any,
+    None,
     Patch(PatchId),
 }
 
-impl Default for Update {
+impl Default for OptPatch {
+    fn default() -> Self {
+        Self::Any
+    }
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum OperationName {
+    Create,
+    Update,
+    List,
+}
+
+impl Default for OperationName {
     fn default() -> Self {
-        Self::No
+        Self::List
     }
 }
 
-#[derive(Default, Debug)]
+#[derive(Debug)]
+pub enum Operation {
+    Create {
```

Comment:
Rename to `Open` along with `OperationName`

Reply:
I was following `rad issue`s lead on this one.  We should change there as well if we do here.

https://github.com/radicle-dev/heartwood/blob/master/radicle-cli/src/commands/issue.rs#L147